starlite/utils/module_loading.py

```diff
@@ -0,0 +1,32 @@
+import sys
+from importlib import import_module
+
+
+def import_string(dotted_path):
+    """
+    Import a dotted module path and return the attribute/class designated by the
+    last name in the path. Raise ImportError if the import failed.
+
+    Based on Django's module_loading code
+
+    """
+
+    def cached_import(module_path, class_name):
+        # Check whether module is loaded and fully initialized.
+        if not (
+            (module := sys.modules.get(module_path))
+            and (spec := getattr(module, "__spec__", None))
+            and getattr(spec, "_initializing", False) is False
+        ):
+            module = import_module(module_path)
+        return getattr(module, class_name)
+
+    try:
+        module_path, class_name = dotted_path.rsplit(".", 1)
+    except ValueError as err:
+        raise ImportError("%s doesn't look like a module path" % dotted_path) from err
+
+    try:
+        return cached_import(module_path, class_name)
+    except AttributeError as err:
```

Comment:
```suggestion
    except AttributeError as e:
```

Very javascripty. Please stick to `e` or `exc`

Reply:
Also please use format strings.